pkg/handler/handler.go

```diff
@@ -174,14 +174,22 @@ func (h handler) deprovision(w http.ResponseWriter, r *http.Request, params map[
 		writeResponse(w, http.StatusBadRequest, broker.ErrorResponse{Description: "invalid instance_uuid"})
 		return
 	}
+
 	var async bool
 	queryparams := r.URL.Query()
```

Comment:
Logged an issue to update the request layer, https://github.com/openshift/ansible-service-broker/issues/303